gradle/test-mocha-js.gradle

```diff
@@ -19,7 +19,9 @@ task testMochaNode(type: NodeTask, dependsOn: [compileTestKotlinJs, installDepen
     if (project.hasProperty("teamcity")) args += ['--reporter', 'mocha-teamcity-reporter']
 }
 
-jsTest.dependsOn testMochaNode
+def legacyjsTestTask = project.tasks.findByName('legacyjsTest') ? legacyjsTest : jsTest
+
+legacyjsTestTask.dependsOn testMochaNode
```

Comment:
same question regarding inconsistency with atomicfu

Reply:
Built-in test runner does not support custom `outputFile` compiler options. These options are used only in legacy backend. So legacy backend tests are running as previously, using custom `testMochaNode` task.